pkgs/desktops/gnome/extensions/impatience/default.nix

```diff
@@ -15,6 +15,14 @@ stdenv.mkDerivation rec {
     glib
   ];
 
+  patches = [
+    # Add gnome-shell 46 support
+    (fetchpatch {
+      url = "https://github.com/timbertson/gnome-shell-impatience/commit/5a14a58e6eb1d95e22fc6795c60c9760d011aa06.patch";
+      hash = "sha256-lyMQjTWJqr+JCQvKeMBC+IV5FH4aLSK61LBYQTo5xuU=";
+    })
+  ];
+
```

Comment:
Sorry there was no tagged version, I've pushed `version-0.5.1` now so you can use that instead.

Reply:
Thank you! I've dropped the commit adding the patch and bumped to 0.5.1.